doc/sphinx/source/using.rst

```diff
@@ -276,6 +276,12 @@ pocl.
               might avoid storing work-item context to memory.
               However, the code bloat is increased with larger
               WG sizes.
+    
+    cbs    -- Use continuation-based synchronization to execute work-items
+              on non-SPMD devices.
+              CBS is expected to work for kernels that 'loops' does not support.
+              For most other kernels it is expected to perform slightly worse.
+              Also enables the LLVM LoopVectorizer.
```

Comment:
Here you could advertise your master's/paper by adding a link to it for people wanting to understand how it works. Which reminds me... if you have any PoCL-comparing/related publications, you can add them to the publications.mak to advertise. I can then make it visible here: http://portablecl.org/publications.html

Reply:
I'll decide which to add there after the IWOCL deadline ;-)